[IMP] Rewrote sentences in the introduction portion for both sections (#923)
odoo/documentation: content/applications/services/helpdesk/overview/forum_and_elearning.rst

   :align: center
   :alt: Overview of a forum’s settings page in Odoo Helpdesk

After clicking *Save*, logged in users can now start their discussions. To keep track of posts,
go to
:menuselection:`Website --> Forum --> Posts`.

.. image:: forum_and_elearning/customer-care-website.png
   :align: center
   :alt: Overview of the Forums page of a website to show the available ones in Odoo Helpdesk

Turn tickets into forum posts by simply clicking *Share on the Forum* on the ticket form.

.. image:: forum_and_elearning/customer-care-share-on-forum.png
   :align: center
   :alt: Helpdesk ticket form setting to select Share on the Forum feature.

eLearning
=========

In addition to a forum, online courses can improve customer experience and save your support team
valuable time. Your customers can find answers themselves through tutorials, and your team
can send links to helpful content rather than writing out lengthy emails.

Set up
------

Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, select a Helpdesk Team, and
enable *eLearning* under Self-Service.

.. image:: forum_and_elearning/customer-care-eLearning-setting.png
   :align: center

>    :alt: Overview of the settings page of a customer care team emphasizing the feature elearning in
>          Odoo Helpdesk

Reviewer: ```suggestion
   :alt: Overview of the settings page of a customer care team emphasizing the eLearning feature in
         Odoo Helpdesk
```

> Click on the external link to create or edit an eLearning channel.

Reviewer: I had to select a "Related channel" first.

.. image:: forum_and_elearning/customer-care-basics-furniture-creation.png
   :align: center
   :alt: Overview of the settings page of an eLearning course emphasizing the feature elearning in
         Odoo Helpdesk

> Once the structure and content of your course are ready, *Publish* by clicking on *Unpublished*.

Reviewer: ```suggestion
Once the structure and content of your course are ready, switch the page to *Published* by clicking on the *Unpublished* toggle button.
```

> .. image:: forum_and_elearning/eLearning-course-published.png

Reviewer: New screenshot needed.

   :align: center
   :alt: View of a course being published for Odoo Helpdesk

> Use the *eLearning app* for an overview of all your courses and to keep track of course
> statistics, attendees, views, reviews, and more.

Reviewer: ```suggestion
Use the *eLearning app* for an overview of all your courses and to keep track of course
statistics, such as the number of attendees, views, reviews, and more.
```

> .. image:: forum_and_elearning/eLearning-app-statistics.png

Reviewer: New screenshot needed.

   :align: center

>    :alt: View of the elearning applications dashboard for Odoo Helpdesk

Reviewer: ```suggestion
   :alt: View of the eLearning applications dashboard for Odoo Helpdesk
```
